frontend/src/routes/Contributors/Contributors.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import PATH from '../../utils/path'
+import Contributors from './Contributors'
+
+export default {
+  title: `routes/${PATH.CONTRIBUTORS}`,
```

Comment:
Storybook fails to load if any *.stories.js file has a `title` ending with a slash or contain consecutive slashes. The console error for Contributors looks like `Uncaught (in promise) Error: Invalid part '', leading to id === parentId ('routes'), inside kind 'routes//contributors/'`.

You can fix this if you change this line to ``title: `routes${PATH.CONTRIBUTORS}`,`` and edit utils/paths.js to not have a trailing slash.

Same for all of the other route stories except Home.

Reply:
Fixed